jerry-core/debugger/debugger.h

```diff
@@ -112,10 +112,9 @@ typedef enum
   JERRY_DEBUGGER_EXCEPTION_STR_END = 18, /**< exception string last fragment */
   JERRY_DEBUGGER_BACKTRACE = 19, /**< backtrace data */
   JERRY_DEBUGGER_BACKTRACE_END = 20, /**< last backtrace data */
-  JERRY_DEBUGGER_EVAL_RESULT = 21, /**< eval result */
-  JERRY_DEBUGGER_EVAL_RESULT_END = 22, /**< last part of eval result */
-  JERRY_DEBUGGER_EVAL_ERROR = 23, /**< eval result when an error is occured */
-  JERRY_DEBUGGER_EVAL_ERROR_END = 24, /**< last part of eval result when an error is occured */
+  JERRY_DEBUGGER_SEND_EVAL = 21, /**< send over the eval result */
```

Comment:
No need to add SEND to the string. Since this is a message SEND is obvious. `JERRY_DEBUGGER_EVAL_RESULT` perhaps sound better.